agent/rpm/Makefile

```diff
@@ -53,8 +53,7 @@ patches:
 .PHONY: tarball
 tarball:
 	mkdir -p ${RPMSRC}
-	git submodule init
-	git submodule update
+	(cd ${AGENT}/../; git submodule init; git submodule update)
```

Comment:
just a question, does this change the working directory and do we need to cd back to the rpm directory?

Reply:
The closing parentheses should prevent the changing of the default working directory from escaping just that command.

Reply:
This is a shell mechanism: executing a command inside parens does it in a subshell, so changing the directory in the subshell does not affect the parent shell at all.